### PR TITLE
Update postprocessing.py

### PR DESCRIPTION
**Pull request by llobetv**
sorte template and template_ind based on their amplitude. 
Phy use this index to find the best channel

### CODE REVIEW

**Comment by llobetv** [comment]
To test:
```
import numpy as np
import pytest
import spikeextractors as se
from spiketoolkit.postprocessing import get_unit_waveforms, get_unit_templates, get_unit_amplitudes, \
    get_unit_max_channels, set_unit_properties_by_max_channel_properties, compute_unit_pca_scores, export_to_phy, \
    compute_unit_template_features
from spiketoolkit.preprocessing import remove_bad_channels
import os
import shutil
from pathlib import Path




@pytest.mark.implemented
def test_export_to_phy():
    folder = 'test'
    if os.path.isdir(folder):
        shutil.rmtree(folder)
    rec, sort = se.example_datasets.toy_example(dump_folder=folder, dumpable=True, duration=10, num_channels=8, seed=0)

    export_to_phy(rec, sort, output_folder='phy')
    rec.set_channel_groups([0, 0, 0, 0, 1, 1, 1, 1])
    export_to_phy(rec, sort, output_folder='phy_group', grouping_property='group', recompute_info=True)
    export_to_phy(rec, sort, output_folder='phy_max_channels', max_channels_per_template=4, recompute_info=True)
    export_to_phy(rec, sort, output_folder='phy_no_feat', grouping_property='group', compute_pc_features=False,
                  recompute_info=True)
    export_to_phy(rec, sort, output_folder='phy_no_amp', compute_amplitudes=False)
    export_to_phy(rec, sort, output_folder='phy_no_amp_feat', compute_amplitudes=False,
                  compute_pc_features=False)
    export_to_phy(rec, sort, output_folder='phy_par', n_jobs=2)

    rec_phy = se.PhyRecordingExtractor('phy')
    rec_phyg = se.PhyRecordingExtractor('phy_group')
    assert np.allclose(rec.get_traces(), rec_phy.get_traces())
    assert np.allclose(rec.get_traces(), rec_phyg.get_traces())
    assert not (Path('phy_no_feat') / 'pc_features.npy').is_file()
    assert not (Path('phy_no_feat') / 'pc_feature_ind.npy').is_file()
    assert not (Path('phy_no_amp') / 'amplitudes.npy').is_file()
    assert not (Path('phy_no_amp_feat') / 'amplitudes.npy').is_file()
    assert not (Path('phy_no_amp_feat') / 'pc_features.npy').is_file()
    assert not (Path('phy_no_amp_feat') / 'pc_feature_ind.npy').is_file()

    sort_phy = se.PhySortingExtractor('phy', load_waveforms=True)
    sort_phyg = se.PhySortingExtractor('phy_group', load_waveforms=True)

    assert np.allclose(sort_phy.get_unit_spike_train(0), sort.get_unit_spike_train(sort.get_unit_ids()[0]))
    assert np.allclose(sort_phyg.get_unit_spike_train(2), sort.get_unit_spike_train(sort.get_unit_ids()[2]))
    assert sort_phy.get_unit_spike_features(1, 'waveforms').shape[1] == 8
    assert sort_phyg.get_unit_spike_features(3, 'waveforms').shape[1] == 4

    rec.set_channel_groups([0, 0, 0, 0, 1, 1, 1, 1])
    recrm = remove_bad_channels(rec, [1, 2, 5])
    export_to_phy(recrm, sort, output_folder='phy_rm', grouping_property='group', recompute_info=True, verbose=True)
    templates_ind = np.load('phy_rm/template_ind.npy')
    assert len(np.where(templates_ind == -1)[0]) > 0  # removed channels are -1


if __name__ == '__main__':
    test_export_to_phy()

```

**Comment by llobetv** [comment]
I didn't change pc feature as their is pc_features_ind, it should be independant of templates_ind.
I didn't see any pb with pc features during manlual inspection

**Comment by llobetv** [comment]
#374

File: spiketoolkit/postprocessing/postprocessing_tools.py [hunk 1]
```diff
@@ -1658,16 +1658,37 @@ def _get_phy_data(recording, sorting, compute_pc_features, compute_amplitudes,
...
 
     # spike_templates.npy - [nSpikes, ] uint32
     spike_templates = spike_clusters
-
+    
     return spike_times, spike_clusters, amplitudes, channel_map, pc_features, pc_feature_ind, \
            spike_templates, templates, templates_ind, similar_templates, channel_map_si, channel_groups, positions
 
 
+def _template_descending_order(recording, sorting, templates, templates_ind):
```
Comment (alejoe91) [inline spiketoolkit/postprocessing/postprocessing_tools.py:1674]: Why do you need `sorting` as argument?

Comment (llobetv) [reply spiketoolkit/postprocessing/postprocessing_tools.py]: you are right, i was a bit too fast i when I create the function

**Comment by llobetv** [comment]
> Hi @llobetv
> 
> Thanks! Let me test that pca are not affected. You can see [here](https://github.com/SpikeInterface/spiketoolkit/blob/master/spiketoolkit/postprocessing/postprocessing_tools.py#L948) that the template indexes might affect the way pc scores are computed, but I need to double check. Will do it tomorrow morning

It should be ok since I change template order after that pca is calculated?

**Comment by alejoe91** [comment]
@llobetv I double checked that the FeatureView is unchanged and the channels are extracted correctly! Thanks again!

**Pull request merged**